### PR TITLE
Removed redudant tesseractRosUtilsToMsg usage

### LINKED ISSUE #49: Duplicate Code: crs_motion_planning::tesseractRosutilsToMsg
What is the purpose of this conversion? It is nearly identical to the one in tesseract_ros2 except one uses rclcpp::Duration and one uses chrono. Is this a bug fix that should be pushed back or did the other conversion just not exist at the time?

### PR DESCRIPTION
**Pull request by marrts**
Resolves #49

### CODE REVIEW

File: crs_motion_planning/include/crs_motion_planning/path_processing_utils.h [hunk 1]
```diff
@@ -37,17 +37,6 @@ namespace crs_motion_planning
 bool parsePathFromFile(const std::string& yaml_filepath,
                        const std::string& waypoint_origin_frame,
                        std::vector<geometry_msgs::msg::PoseArray>& raster_strips);
-
```
Comment (mpowelson) [inline crs_motion_planning/include/crs_motion_planning/path_processing_utils.h:40]: I think you can get rid of line 22 now. I think this is the only one that depends on tesseract
```
#include <tesseract_common/types.h>
```

File: crs_motion_planning/src/path_processing_utils.cpp [hunk 2]
```diff
@@ -70,35 +70,6 @@ bool crs_motion_planning::parsePathFromFile(const std::string& yaml_filepath,
   return true;
 }
 
-void crs_motion_planning::tesseractRosutilsToMsg(trajectory_msgs::msg::JointTrajectory& traj_msg,
```
Comment (mpowelson) [inline crs_motion_planning/src/path_processing_utils.cpp:73]: It looks like you don't need tesseract here either anymore. You should be able to remove `  tesseract::tesseract_common` from the target_link_libraries

**Review by mpowelson — CHANGES_REQUESTED** [review]
Looks good. You'll need to run clang formatting. Also, if you can remove the tesseract dependency that would greatly be preferred.

**Review by mpowelson — APPROVED** [review]

**Comment by mpowelson** [comment]
When CI passes we can merge.

**Comment by mpowelson** [comment]
I'm not sure why this failed.

**Comment by marrts** [comment]
I was afraid this might happen, something to do with the tesseract linked library in path_processing_utils. It also failed on my machine on my first attempt to build crs_gui.

**Comment by mpowelson** [comment]
Do you want me to look into this?

**Comment by marrts** [comment]
Yeah, I think it has to do with crs_gui dependencies, so you'd probably be better equipped to look into it. I also found it odd that I had to add iterative_spline_parameterization to the crs_gui CMakeLists since it shouldn't need to use that anywhere, maybe these two issues are related.

**Comment by mpowelson** [comment]
Yeah that doesn't sound good. We must have some CMake gremlins.

**Comment by mpowelson** [comment]
I pushed some changes that will let it build at this point. Jorge has some more fixes in his PR that cleans up a few other things, but hopefully this will pass.

**Comment by marrts** [comment]
Seems like this should be good to merge unless there are any objections

**Pull request merged**